pkg/trait/gc.go

```diff
@@ -109,7 +113,12 @@ func getOldGenerationResources(e *Environment) ([]unstructured.Unstructured, err
 		return nil, err
 	}
 
-	selector, err := labels.Parse(fmt.Sprintf("camel.apache.org/integration=%s,camel.apache.org/generation,camel.apache.org/generation notin (%d)", e.Integration.Name, e.Integration.GetGeneration()))
+	selectors := []string{
+		fmt.Sprintf("camel.apache.org/integration=%s", e.Integration.Name),
+		fmt.Sprintf("camel.apache.org/generation,camel.apache.org/generation notin (%d)", e.Integration.GetGeneration()),
+	}
```

Comment:
@astefanutti I had to split the line to fix lint findings, can you check it it is still ok ?

Reply:
@lburgazzoli I knew that line was way too long 😉! While we are at it, you could split the second selector as well for consistency. Otherwise LGTM.

Reply:
I will

Reply:
done